genomedata/__init__.py

```diff
@@ -335,10 +307,9 @@ def add_track_continuous(self, trackname):
             raise NotImplementedError("""Adding tracks is only supported \
 for archives created with Genomedata version 1.2.0 or later.""")
 
-        if self._isfile:
-            add_trackname(self, trackname)
+        self._chromosomes._add_trackname(trackname)
```

Comment:
Why are some `ChromosomeList` methods underscored and others not? The whole class is "private", right?

Reply:
There isn't a good reason. The class is private anyway. I think this is just a hold-over naming scheme from before that should have been changed.